Added vagrant file

Paul-Stefan opened the pull request:
Added vagrant file for easy setup. 
Still a couple of problems

Paul-Stefan committed:
Added vagrant file

Added vagrant file for easy setup. 
Still a couple of problems

Paul-Stefan committed:
added vagrant file

ckritzinger commented:
For some reason, there are a ton of files that should be in the root now added to the bin/ directory.

Please remove those.  The only file you should need to add at this point is the Vagrantfile

Paul-Stefan committed:
added my notes

added notes describing each step I took just for refrence (might come in handy to know what helped solve a problem)

Paul-Stefan committed:
added text file

Paul-Stefan commented:
Hi there.

Sorry that was a complete fail from my part. Was rushing to get it uploaded
before leaving and messed up and decided to fix it tomorrow morning.

Regards


On 05 Jan 2017 16:26, "ckritzinger" <notifications@github.com> wrote:

> For some reason, there are a ton of files that should be in the root now
> added to the bin/ directory.
>
> Please remove those. The only file you should need to add at this point is
> the Vagrantfile
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ckritzinger/turtology/pull/1#issuecomment-270655804>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AXMGWRxXTBbpaS7kPP-SdTKiWjFOjzwGks5rPP2wgaJpZM4LbvDQ>
> .
>
